docs/site/Model.md

```diff
@@ -190,29 +190,15 @@ class Product extends Entity {
 }
 ```
 
-However, the model decorator in LoopBack 4 is not exactly the same as what it is
-in LoopBack 3. For example, in
-[lb3](https://loopback.io/doc/en/lb3/Model-definition-JSON-file.html) we can
-pass in a model definition syntax in the model decorator, such as properties,
-options, relation etc. But not all these entries are available in lb4 model
-decorator:
+{% include note.html content="If you used LB3 before, the model decorator in LoopBack 4 is not exactly the same as what it is
+in LoopBack 3. For example, properties and relation cannot be defined through the model decorator. Please check the following section for available entires." %}
```

Comment:
Can you change LB3 in the line before to "LoopBack 3" and change the second one to LB3?
`relation` -> `relations`
`entires` -> `entries`